client/src/types/regions.ts

```diff
@@ -0,0 +1,22 @@
+export enum Regions {
+    abruzzo = "ABRUZZO",
+    aosta_valley = "AOSTA VALLEY",
+    apulia = "APULIA",
+    basilicata = "BASILICATA",
+    calabria = "CALABRIA",
+    campania = "CAMPANIA",
+    emilia_romagna = "EMILIA-ROMAGNA",
+    friuli_venezia_giulia = "FRIULI-VENEZIA GIULIA",
+    lazio = "LAZIO",
+    liguria = "LIGURIA",
+    lombardy = "LOMBARDY",
+    marche = "MARCHE",
+    molise = "MOLISE",
+    piedmont = "PIEDMONT",
+    sardinia = "SARDINIA",
+    sicily = "SICILY",
+    trentino_south_tyrol = "TRENTINO-SOUTH TYROL",
+    tuscany = "TUSCANY",
+    umbria = "UMBRIA",
+    veneto = "VENETO",
```

Comment:
Nice! I think we have to discuss with Davide how these are being sent to the backend. For example, for Friuli-Venezia Giulia, only the word "Friuli" returned something from ninja-api. I'm not sure how Davide has set up the endpoints yet.